### ISSUE TITLE
Stars count next to the list item

### ISSUE THREAD

**Issue by dospolov** [vitejs/awesome-vite#313]
Github Stars count is an important criteria for choosing plugins. 
What do you think about placing a label with a Stars count next to each list item?

**Comment by Jogai** [comment]
Personally I dont pay attention to github stars. I bet all repo's that were once popular, but are now out of fashion still have a lot of stars.

That said, badgen is made for this so its never out of date.

`![awesome-vite-stars](https://flat.badgen.net/github/stars/vitejs/awesome-vite)`

Resulting in:

![awesome-vite-stars](https://flat.badgen.net/github/stars/vitejs/awesome-vite)

**Comment by dospolov** [comment]
Cool,
I can send a PR with badges if you think that might be relevant for others.

**Comment by Jogai** [comment]
Well, I'm not a maintainer here so I cant really say.

**Comment by Scrum** [comment]
I think this is a somewhat dubious choice based on the stars, for example, an older package may have more stars due to time, but not have support for a long time than a young package that has all the advantages that has an insignificant number of stars.

**Comment by utamori** [comment]
I think the last commit is a good indicator that the project is being actively maintained.　

`![](https://flat.badgen.net/github/last-commit/vitejs/awesome-vite)`
![](https://flat.badgen.net/github/last-commit/vitejs/awesome-vite)

**Comment by millette** [comment]
Showing both (stars/last commit) would be a good indicator.

**Comment by HugoImaios** [comment]
Indeed, this awesome list a very dense, and contain project wich haven't been updated for years. The "why not both" of @millette seems a nice to have :+1: 
@Scrum what would you think about it ?
